packages/p2p/src/components/orders/order-details/order-actions-block.jsx

```diff
@@ -34,11 +36,34 @@ const OrderActionsBlock = ({ cancelPopup, order_details, showPopup }) => {
             title: localize('Cancel this order?'),
             message: localize('If you have paid, please do not cancel the order.'),
             confirm_text: localize('Cancel this order'),
+            has_cancel: true,
+            cancel_text: localize('Do not Cancel'),
             onClickConfirm: cancel,
         };
         showPopup(options);
     };
-
+    const compalin = () => {
+        const options = {
+            title: localize('Something went wrong?'),
+            message: (
+                <Localize
+                    i18n_default_text='If you can not resolve a dispute with the other party, please email
+                <0>p2p-support@deriv.com</0>. Describe your situation and include your order 
```

Comment:
We can use `context.email_domain` instead of hardcoding `deriv.com`, please see `p2p/src/components/my-ads/my-ads.jsx` on line 73.

Reply:
updated